Objects/Converters/ConverterAutocadCivil/ConverterAutocadCivilShared/ConverterAutocadCivil.Geometry.cs

```diff
@@ -57,12 +57,9 @@ public Point PointToSpeckle(Point2d point, string units = null)
 
   public Point3d PointToNative(Point point)
   {
-    var _point = new Point3d(
-      ScaleToNative(point.x, point.units),
-      ScaleToNative(point.y, point.units),
-      ScaleToNative(point.z, point.units)
-    );
-    var intPt = ToInternalCoordinates(_point);
+    var scaleFactor = Units.GetConversionFactor(point.units, ModelUnits);
```

Comment:
An optional improvement, but if we add `double? conversionFactor = null` as an optional parameter and calculate this at the mesh level, you may be able to squeeze out a bit more performance here as well (instead of running `GetConversionFactor` for every point)

Reply:
Very true, I imagine we're using this function in quite a few places, so there's definitely more performance on the cards.

For tin surfaces tho, this wasn't in the hot path, so won't give us a tangeble performance improvement. And I wanted to keep this pr from bloating into other conversions.

For the dui3, we should make sure we don't carry this across, and try and perform fewer unit lookups (at least for the heavy objects like mesh, brep, and curves where we expect potentially hundreds of thousands of points.)

Reply:
OOS